src/main/java/org/opentripplanner/graph_builder/module/osm/OsmDatabase.java

```diff
@@ -1115,14 +1115,18 @@ private void processPublicTransportStopArea(OSMRelation relation) {
         relationsById.containsKey(member.getRef())
       ) {
         platformAreas.add(relationsById.get(member.getRef()));
-      } else if (
-        "node".equals(member.getType()) &&
-        nodesById.containsKey(member.getRef()) &&
-        nodesById.get(member.getRef()).isEntrance()
-      ) {
-        platformNodes.add(nodesById.get(member.getRef()));
+      } else if ("node".equals(member.getType()) && nodesById.containsKey(member.getRef())) {
```

Comment:
Can you move these two new checks into methods of `OSMWithTags` or `OSMNode` so we have an explicit name and a bit of documentation for them?

Reply:
Can you please replace this piece of stringly typed code with a method `isNodeType()`?

If you want to be extra nice you can replace the type variable of a `OSMRelationMember` with an enum, since there are only three values possible: https://github.com/opentripplanner/OpenTripPlanner/blob/3048c792d83ad6d2bdfcf7946d1320aa9c757358/src/main/java/org/opentripplanner/openstreetmap/OpenStreetMapParser.java#L87-L96

There is lots and lots of repeated versions of this which we can also replace.

Reply:
Good idea - I will do that.